src/components/photopage/profile.js

```diff
@@ -28,7 +27,9 @@ const styles = theme => ({
 		fontFamily: ['Chivo', 'sans-serif'],
```

Comment:
Same here. We could replace this with theme.typography.fontFamily

Reply:
you are right!